ui/css/treeherder-job-buttons.css

```diff
@@ -73,8 +74,14 @@
   transform: scale(1.7, 1.7);
 }
 
+@keyframes fadein {
+  from { opacity: 0; }
+  to   { opacity: 1; }
+}
+
 .filter-shown {
   display: inline-block;
+  animation: fadein 0.1s;
```

Comment:
I *love* this fadein for group expand/collapse! It really helps the user better see what is happening especially with 20+ collapsed chunks. I had to increase the duration though to something like `0.5s` on my own machine to really see it. At 0.1 it looked the same as stage (instant). At least on my OSX MacBook Pro w/Nightly.

On Chrome this transition appears to cause the entire job block to fadein, on initial page load. I'm not sure that is a big deal but I have a feeling when Treeherder polls for new jobs the whole job block may 're-fadein on Chrome. I think I recall this when I was attempting similar last year with Show/Hide excluded jobs (before they were changed to reload the entire page).

Maybe @wlach if you're running vagrant you can sit on the page for a minute or two in Chrome to see what it does during ingestion/polling?